plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -169,6 +170,7 @@ def setup(self):
         self.ui.serverComboBox.lineEdit().setPlaceholderText("enter server address or leave empty to use default")
         self.ui.fetchServerInfoButton.setIcon(self.icon("refresh-icon.png"))
         self.ui.segmentationButton.setIcon(self.icon("segment.png"))
+        self.ui.contourButton.setIcon(self.icon("icon_contour.svg"))
```

Comment:
Can we rename this file to contour.svg to make all same similar.. already they are in Icons folder 

Also add this in CMAKE file.. to be part of release plugin